src/providers/twitch/api/Helix.hpp

```diff
@@ -340,6 +340,19 @@ enum class HelixDeleteChatMessagesError {
     Forwarded,
 };
 
+enum class HelixAddChannelModeratorError {
+    Unknown,
+    UserMissingScope,
+    UserNotAuthenticated,
+    UserNotAuthorized,
+    UserIsVIP,
```

Comment:
Naming of this error is too similar to `UserMissingScope`, so we use `User` for both describing the User who is running the command and the User who is the target of the API request

Reply:
Changed to `TargetIsVIP`